tasks/main-tang.yml

```diff
@@ -1,81 +1,36 @@
 ---
 - name: Ensure tang is installed
   package:
-    name: "{{ __nbde_server_package_list }}"
+    name: "{{ __nbde_server_packages }}"
     state: present
 
-
-- name: Ensure tang is enabled and started
-  when: state is not defined or state != "stopped"
-  service:
-    name: "{{ __nbde_server_service }}"
-    enabled: yes
-    state: started
-
-
-- name: Ensure tang is enabled but stopped
-  when: state is defined and state == "stopped"
-  service:
-    name: "{{ __nbde_server_service }}"
-    enabled: yes
-    state: stopped
-
-
 - name: Ensure keys are rotated
   when: nbde_server_rotate_keys
```

Comment:
to avoid the bare variable problem, this should be `when: nbde_server_rotate_keys | bool`